merge.py

```diff
@@ -0,0 +1,82 @@
+from SPARQLWrapper import get_sparql_dataframe
+# for now, import SPARQLWrapper as well, to redefine __agent__:
+import SPARQLWrapper 
+
+import geopandas as gpd
+import os
+import pandas as pd
+import urllib.request
+
+USER_AGENT = "github.com/highsource/verbundkarte/0.0.1"
+CACHE_DIR = '.cache'
+OUT_DIR = 'out'
+# WFS Service: https://sgx.geodatenzentrum.de/wfs_vg250-ew?REQUEST=GetCapabilities&SERVICE=WFS
+VG250_URL = 'https://sgx.geodatenzentrum.de/wfs_vg250-ew?REQUEST=GetFeature&SERVICE=WFS&VERSION=2.0.0&outputFormat=json&typeNames=vg250-ew:vg250_krs'
+AUTHORITIES_FILE = 'data/authorities.csv'
+ASSIGNMENTS_FILE = 'data/assignments.csv'
+CACHED_VG250_PATH = os.path.join(CACHE_DIR, 'vg250_krs.json')
+VG250_GEOFAKTOR_LAND_MIT_STRUKTUR = 4
+ENHANCED_AUTHORITIES_PATH = os.path.join(OUT_DIR, 'authorities_enhanced.geojson')
+
+def assert_dir_exists(path):
+	if not os.path.exists(path):
+		os.mkdir(path)
+
+def download_if_not_chached(path, url):
+	if not os.path.exists(path):
+		urllib.request.urlretrieve(url, path)
+
+def setup():
+	assert_dir_exists(OUT_DIR)
+	assert_dir_exists(CACHE_DIR)
+	download_if_not_chached(CACHED_VG250_PATH, VG250_URL)
```

Comment:
According to the change in the function before
```suggestion
	download_if_not_cached(CACHED_VG250_PATH, VG250_URL)
```